tools/admin/batch_invite.py

```diff
@@ -0,0 +1,36 @@
+from github import Github
+import os
+
+token = os.environ.get("GH_TOKEN")
+organization_name = os.environ.get("ORG_NAME", "instructlab")
+
+def main():
+
+    with open("../names.txt") as names_file:
+        names = names_file.readlines()
+
+    total = 0
+
+    for name in names:
+        try:
+            gh = Github(token)
+            org = gh.get_organization(organization_name)
+            ghuser = gh.get_user(name.strip())
+            ghlablovers = org.get_team_by_slug(os.environ.get("TEAM_NAME", "Labrador-Lovers"))
```

Comment:
The lab lovers team was just removed! I am wondering if we still would need batch invite/remove scripts anymore since the project is now public? I would lean to removing the admin folder and just add the batch comment script.

Reply:
It looks like this still needs resolution @jjasghar

Reply:
Done, removed the file(s).